src/api/checkout.js

```diff
@@ -24,4 +24,18 @@ app.post('/', (req, res) => {
     });
 });
 
+app.post('/longterm', (req, res) => {
+    addAction('NEW_LONGTERM_CHECKOUT', {
```

Comment:
>Added?

No, they were not added. Please click the link @Cafferoq provided.

It looks like you've added "functional" tests that only add actions, but they should be calling the server with a mock client in order to hit this endpoint.